cpp/server/ct-server.cc

```diff
@@ -167,6 +168,7 @@ static const bool sign_dummy =
     RegisterFlagValidator(&FLAGS_tree_signing_frequency_seconds,
                           &ValidateIsPositive);
 
+
```

Comment:
I have refrained from doing this, because of, well, the whole rest of the file. :smiley: